frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx

```diff
@@ -131,14 +132,14 @@ class CreateVMWizardComponent extends React.Component<CreateVMWizardComponentPro
   };
 
   getWizardTitle() {
-    const { isCreateTemplate, isProviderImport } = this.props;
+    const { isCreateTemplate, isProviderImport, userTemplateName } = this.props;
     if (isCreateTemplate) {
       return CREATE_VM_TEMPLATE;
     }
     if (isProviderImport) {
       return IMPORT_VM;
     }
-    return CREATE_VM;
+    return userTemplateName ? `${CREATE_VM} from ${userTemplateName} template` : CREATE_VM;
```

Comment:
yes we could include
template as the first field and a read only input with the name of the template

Reply:
IMO I quite like that it is part of the header since it is an identifying info of the wizard and you can see it from all the tabs. Can we try to play with the idea of a header a bit longer?

what about 
`Create VM from vm-template-example`

or 

`Create Virtual Machine from vm-template-example`

or some kind of highligting.

Nevertheless, we still could opt to not to do anything and keep the header as it is.

Reply:
I'm ok with the second option

Reply:
OK. Second option it is: `Create Virtual Machine from vm-template-example`